README.md

```diff
@@ -64,14 +65,15 @@ podman unshare chown 0:0 -R .
 ```
 To access the container once running, go to [http://localhost:6080/vnc.html](http://localhost:6080/vnc.html).
 
-If you want to program the FPGA from the container, lets find out which bus and device our Arty is on:
+If you want to program the FPGA from the container, let's find out which bus and device our Arty is on:
```

Comment:
Maybe worth fixing the people here too: "you" want to program the FPGA, but it's "our" Arty! I'd be lazy and say "device the Arty is on"

Reply:
Good point, changed!